DOCS.md

```diff
@@ -927,6 +936,30 @@ login({appState: JSON.parse(fs.readFileSync('appstate.json', 'utf8'))}, (err, ap
 
 ---------------------------------------
 
+<a name="setMessageReaction"></a>
+### api.setMessageReaction(reaction, messageID[, callback])
+
+Sets reaction on message
+
+__Arguments__
+
+* `reaction`: A string contains `emoji`, `emoji shortcut`, `emoji in unicode` or left `empty string` for delete reaction (look down for list of supported emojis)
+* `messageID`: A string representing the message ID.
+* `callback(err)` - A callback called when sending the reaction is done.
+
+__Supported Emojis__
+
+* 😍 - Unicode: `\uD83D\uDE0D`, Shortcut: `:heart_eyes:`
+* 😆 - Unicode: `\uD83D\uDE06`, Shortcut: `:laughing:`
+* 😮 - Unicode: `\uD83D\uDE2E`, Shortcut: `:open_mouth:`
+* 😢 - Unicode: `\uD83D\uDE22`, Shortcut: `:cry:`
+* 😠 - Unicode: `\uD83D\uDE20`, Shortcut: `:angry:`
+* 👍 - Unicode: `\uD83D\uDC4D`, Shortcut: `:thumbsup:`
+* 👎 - Unicode: `\uD83D\uDC4E`, Shortcut: `:thumbsdown:`
```

Comment:
In my opinion, it would be better if reaction names shortcuts were unified with the names of the reactions for Facebook posts, i.e.:
* :love:
* :haha:
* :wow:
* :sad:
* :angery:
* :like:
* :dislike: